### PR TITLE
feat: Names search case insensitive

### LINKED ISSUE #817: Names search case insensitive
**Acceptance criteria**:
- Names search is case insensitive
- The new /v3 endpoint is used

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #817 

## Demo

firefox_sTUua1fzTr

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] make search case insensitive

### CODE REVIEW

**Review by Liubov-crypto — APPROVED** [review]
LGTM

File: src/components/SearchNamesTable.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
             {{ name.name }}
           </app-link>
         </td>
-        <td>{{ name.status }}</td>
+        <td>{{ name.status ? 'Active' : 'Expired' }}</td>
```
Comment (michele-franchi) [inline src/components/SearchNamesTable.vue:28]: Name status is missing

**Review by michele-franchi — CHANGES_REQUESTED** [review]
All the names have an "Expired" status in the search results page, because the "status" property is not present anymore.

File: src/components/SearchNamesTable.vue (hunk 1 above)
Comment (janmichek) [reply src/components/SearchNamesTable.vue]: Right, fixed.

File: src/utils/format.js [hunk 2]
```diff
@@ -96,16 +96,6 @@ export function formatDecodeByteArray(bytesArray) {
   return String.fromCharCode(...bytesArray)
 }
 
-export function formatNameStatus(name) {
```
Comment (janmichek) [inline src/utils/format.js:99]: I removed this logic as there is no way to determine whether the name is in auction or not. It was not even possible before.

**Review by michele-franchi — APPROVED** [review]
LGTM

**Pull request merged**